daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc/Test.hs

```diff
@@ -33,8 +33,8 @@ import qualified Text.XML.Light as XML
 
 
 -- | Test a DAML file.
-execTest :: [FilePath] -> Maybe FilePath -> Compiler.Options -> IO ()
-execTest inFiles mbJUnitOutput cliOptions = do
+execTest :: [FilePath] -> Bool -> Maybe FilePath -> Compiler.Options -> IO ()
```

Comment:
We try and avoid `Bool` arguments. See elsewhere for where we've lifted these into a type.

Reply:
@akalburgi That's the thing because of which I once put you on a PR out of the blue. If you grep for a line containing both `newtype` and `Bool`, you'll find a good way to deal with this.

Reply:
@martin-drhu-da  i think it is `type Vat = Bool ` no ?

Reply:
@akalburgi I meant something along
https://github.com/digital-asset/daml/blob/0b87c4ceaec14809fb0de37e1af950dd74540717/daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs#L224

Reply:
Ok that makes sense after reading a little bit about newtype vs type keywords